docs/learn/learn-async-backing.md

```diff
@@ -7,104 +7,317 @@ keywords: [parachains, backing, parablock, perspective parachains, unincluded se
 
-Bundles of state transitions represented as blocks may be processed similarly. In the context of
-Polkadot, it aims to increase the throughput of the entire network.
+:::info Learn about Parachain Consensus
+
+To fully follow the material on this page, it is recommended to be familiar with the primary stages
+of the [Parachain Protocol](./learn-parachains-protocol.md).
+
+:::
+
+## Synchronous Backing
+
+Before diving into asynchronous backing, it is important to understand what synchronous backing is
+and what its main limitations are.
 
-:::info
+:::info What is backing?
 
-In order to realize which exact processes this upgrade improves, it is recommended to be familiar
-with the stages of parablock validation, which you may [read here](./learn-parachains-protocol.md).
+**Backing** refers to the process in which a parablock is verified by a subset of validators who
+determine the block’s validity. It is an important step in the validation process for parablocks, as
+it is the first line of defense in ensuring valid state transitions. Validators who back the
+parablock put their stake at risk, and if the block is later found invalid, those validators are
+slashed.
 
 :::
 
-Asynchronous backing enables logical pipelining over the parablock
-[**generation**](https://wiki.polkadot.network/docs/learn-parachains-protocol#collators),
-[**backing**](https://wiki.polkadot.network/docs/learn-parachains-protocol#parachain-phase), and
-[**inclusion**](https://wiki.polkadot.network/docs/learn-parachains-protocol#inclusion-pipeline)
-processes. A parablock may be at different stages, but multiple parablocks can be processed
-simultaneously (in parallel) if needed. Most notably, parablock N + 1 can be generated and backed
-while its predecessor, parablock N, is undergoing inclusion on the relay chain. Processes can occur
-while their ancestors are included on the relay chain.
+In synchronous backing, parablock generation is tightly coupled to the relay chain's progression on
+a one-to-one basis. Every parablock must be generated and backed within a relay-chain block
+(six-second window), and (if successfully backed) it will be included in a relay-chain block (often
+referred to as the **relay parent** as the parablock anchors itself to it) after an additional six
+seconds. Thus, a parablock can be produced every 12 seconds because a new parablock can be produced
+after including the previous one. Note [candidate receipts](#candidate-receipt) and not the
+parablocks themselves are included in relay parents.
+
+### Sync Backing Mechanics
+
+The diagram below shows the pipelining table for synchronous backing.
+
+![sync-backing-pipeline](../assets/sync-backing-pipeline.png)
+
+Backing (B) and inclusion (I) of each parablock happen on different relay chain blocks, and new
+parablocks are generated once the previous parablock has been included.
+[Contextual execution](#contextual-execution) is driven by the latest included parablock in the most
```

Comment:
contextual execution is an old name for async backing, so it should not be used

Reply:
Hi! The term has already passed 3 rounds of review, and it is mentioned in the PBA material: https://polkadot-blockchain-academy.github.io/pba-book/polkadot/async-backing-shallow/page.html#async-backing-another-look (shallow dive async backing)

Reply:
will change the term to something else, like "parablock building context". Is that ok @ordian ?

Reply:
I don't see any mention of "Contextual execution" on the PBA page you linked, there's only a mention of context.

Quoting from https://github.com/paritytech/polkadot/issues/3779:
> I previously referred to this as "Contextual Execution", but I'd prefer to rename it to "Asynchronous Backing", which is the goal, rather than "contextual execution" which doesn't describe the purpose very well and is fact an implementation detail.

Reply:
Deferring to @rphmeier on this